Parsing.py

```diff
@@ -0,0 +1,22 @@
+def parsing(bed_file):
```

Comment:
It's better to name functions as infinitive form without `to`. `parse()`, `collect()`, `check()`